src/MSBuild.Abstractions/MSBuild.Abstractions.csproj

```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="Microsoft.Build.Locator" Version="$(MicrosoftBuildLocatorVersion)" />
     <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" ExcludeAssets="runtime" />
     <PackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
why is this necessary?

Reply:
https://github.com/advisories/GHSA-5crp-9r3c-p9vr

Reply:
n/m I think what you're saying is why is it necessary given the change to `Versions.props` 👍🏻 will remove.